### DISCUSSION TITLE
TFTP and user management  issue

### DISCUSSION THREAD

**Discussion by keerthi-varuman** [ltsp/ltsp#452] [General] [answered]
**TFTP**
sometimes when I re-boot the LTSP Client I am getting **TFTP** time out.
what needed to be done for that....?

I have enabled all this in `/etc/dnsmasq.conf`

```
resolv-file=/etc/resolv.conf
enable-tftp
tftp-root=/srv/tftp
```  
**User management**
when I want to shutdown ltsp client I am getting a message `A stop job is running for User Manager for UID 1016 (1min 39 /2min)`
it is taking 2 min every time to shut down.
please guide me to fix those issues 

Thank you

**Comment by alkisg** [comment]
For the TFTP, what's the output of this command?
grep -r bind-interfaces /etc/dnsmasq*

For the shutdown process, which process is it? It doesn't sound related to LTSP.

**Comment by keerthi-varuman** [comment]
Here is the result of  grep -r bind-interfaces /etc/dnsmasq*

```
root@deepanam.av:~# grep -r bind-interfaces /etc/dnsmasq.conf 
#bind-interfaces
```

**Comment by alkisg** [comment]
Does the tftp timeout happen with all clients, or with just a specific one?
Can you take a photo of its screen when the problem happens?

**Comment by keerthi-varuman** [comment]
Getting TFTP error randomly on random clients, and when we start the server and client on next day all works fine I have no idea where could be the issue. 

Please check the attached image 

![tftp](https://user-images.githubusercontent.com/75841376/116071775-ebc35e80-a6ab-11eb-84f2-c20a28c240f6.jpg)

**Comment by alkisg** [comment]
I converted this issue into a discussion as it's not a bug in the LTSP code.

1. Is your LTSP server IP actually 192.168.5.231, or is that the wrong IP?
2. When this happens in a client, can you boot any other client after that? Or do all clients stop booting then, until you reboot the server?
3. When this happens, if you run `journalctl -fu dnsmasq` on the server, do you see the client TFTP requests?
4. When this happens, try this command from any booted client, to see if the TFTP server is still working or not:

```shell
busybox tftp -g -r /ltsp/ltsp.ipxe -l /tmp/ltsp.ipxe 192.168.5.231
```

**Reply by keerthi-varuman** [reply]
1. The 192.168.5.231 is the correct IP that I am getting from the DHCP server.
2. I tried reboot another client same time and that works fine.
3. I didn't try I need to try when I am getting the problem again. because the problem is fixed when we started the server nextday.
4. i will try this also next time and let you know the error's

**Comment by keerthi-varuman** [comment, marked as answer]
i managed to fix it was configuration issue
